lib/dependencies/LocalModulesHelpers.js

```diff
@@ -40,4 +20,29 @@ function lookup(parent, mod) {
 	}
 
 	return path.join("/");
+};
+
+class LocalModulesHelpers {
+
+	static addLocalModule(state, name) {
+		if(!state.localModules) state.localModules = [];
+		var m = new LocalModule(state.module, name, state.localModules.length);
+		state.localModules.push(m);
+		return m;
+	}
+
+	static getLocalModule(state, name, namedModule) {
+		if(!state.localModules) return null;
+		if(namedModule) {
+			// resolve dependency name relative to the defining named module
+			name = lookup(namedModule, name);
+		}
+		for(var i = 0; i < state.localModules.length; i++) {
+			if(state.localModules[i].name === name)
+				return state.localModules[i];
+		}
+		return null;
+	}
 }
+
+module.exports = LocalModulesHelpers;
```

Comment:
> Making it a class that should never be instantiated just for attaching static methods to it will just make a (distant future?) potential migration to ES modules harder for no benefit.

What could be the possible problem with that? Can you elaborate more about it, please? This is a helper and it's not exposed. So that we don't have risks with this approach.

If this is something not aligned with the project improvement, should be documented somewhere, not only in our minds. If we have this in some place should be good make it accessible for everyone who's want to contribute.

This will help a lot in reviews and will build how to handle with similar code in webpack codebase. 

@sokra @Kovensky @TheLarkInn